README.md

```diff
@@ -74,43 +74,31 @@ Examples of `Build` resource using the example strategies installed by default.
 ## Try it!
 
 * Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster.
-* Clone this repository from GitHub at the v0.3.0 tag:
+* Install [Tekton](https://cloud.google.com/tekton) `v0.20.1`:
+    ```bash
+    $ kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.20.1/release.yaml
+    ```
 
-  ```bash
-  $ git clone --branch v0.3.0 https://github.com/shipwright-io/build.git
-  ...
-  $ cd build/
-  ```
-
-  _Coming soon - install Shipwright Build via kubectl!_
-
-* Install [Tekton](https://cloud.google.com/tekton) by running [hack/install-tekton.sh](hack/install-tekton.sh), it installs v0.20.1.
+* Install Shipwright build controller `v0.3.0`:
 
   ```bash
-  $ hack/install-tekton.sh
+  $ kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.3.0/release.yaml
```

Comment:
In my opinion this call should be installing also the strategies. So that there is no need for a second `kubectl` before starting to create Build/BuildRuns. @ImJasonH what do you think? I could enhance this into the existing `release.yaml` that you introduced in CI?

Reply:
I don't feel strongly either way. Having to separately install strategies makes it clear they're optional/pluggable.

Even though it requires extra steps, those steps might be worth making the point.

Including the strategies in release.yaml insinuates a promise that they'll be supported to the same level as the core controller components.

Reply:
I think strategies are a promise. My opinion is that users should expect to get by default a set of strategies in order to be able to Build with kaniko/buildah/buildpacks. I think what we need to make clear that is optional is the ability for users to customize their own strategies, which we have 0 docs so far.

Reply:
Thanks! I suggest to separate them as two steps, because:
- Reminder the user the build strategies are optional, and can choose what you want to install, and maybe some users or admins just need to install one or two build strategies, no need to install them all
- build strategy is a CR from kube level. the all resources in release.yaml are CRD or configuration related resources
- In future, we may have many buildstrategies, we cannot know what buildstrategy the new user want to try, so in this `Try It`, is just one sample to give a quick start.

Reply:
I tend to also not want to include the build strategies in the release.yaml. But, build strategies will eventually evolve as well if we extend that CRD. Then, the current build strategies might not be installable in the latest release. So, maybe we need to ship another file with the release, like sample_strategies.yaml?

Reply:
I do see a value in installing all the strategies at once. Users should only try to understand which strategy to use , e.g. `kaniko`  vs `buildpacks`  and assume that for whatever strategy they choose, the right resource will be in place. 

If in the future we extend our strategies, as we did with the `ko` one, then also this one needs to be installed.

I like @SaschaSchwarze0 proposal, so we can introduce a new file that goes into the release assets that gives you:
- only clusterbuildstrategies, so that users dont need to deal with a namespace
- kaniko cbs
- buildah cbs
- all buildpacks cbs
- ko cbs

and that can be easily install as a second step, for example:

```sh
kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.3.0/default_strategies.yaml
```

Reply:
> the current build strategies might not be installable in the latest release

It is a good point!

So the `default_strategies.yaml` is also compatible with the `release.yaml`

So... if all agree, let me add this file manually for the release `v0.3.0`, then let us automate it in the future release?

Reply:
Hi all,

I provided a `default_strategies.yaml` file manually in the v0.3.0 release: https://github.com/shipwright-io/build/releases/download/v0.3.0/default_strategies.yaml

And change the command to apply this file directly.

All tests passed, please review again.
Thanks!

Reply:
Nice. I am not fully up-to-date with our release automation. But I think you can extend [release.sh](https://github.com/shipwright-io/build/blob/master/hack/release.sh) to create this yaml file as part of the release process (I guess using `cat` is good enough). And then update the [release workflow](https://github.com/shipwright-io/build/blob/master/.github/workflows/release.yaml#L56-L65) to also attach the file. And there, the step to attach the release-debug.yaml is already missing.

Reply:
Yes, I agree it should be automated for future release.

I suggest for release v.0.3.0, we add it manually. And later, we use another PR to do the automation separately.

Reply:
@zhangtbj is fine with me, but feels not correct. Adding it after the facts manually for v0.3.0 is completely fine. But, this PR now adds documentation that references it. Imo the same PR should also make sure that the file is present for future releases.